attachment_swift/models/ir_attachment.py

```diff
@@ -85,12 +85,16 @@ def _get_swift_connection(self):
         host = os.environ.get('SWIFT_AUTH_URL')
         account = os.environ.get('SWIFT_ACCOUNT')
         password = os.environ.get('SWIFT_PASSWORD')
-        tenant_name = os.environ.get('SWIFT_TENANT_NAME')
+        project_name = os.environ.get('SWIFT_TENANT_NAME'):
+        _logger.warning(
+            "SWIFT_TENANT_NAME is deprecated and "
+            "must be replaced by SWIFT_PROJECT_NAME"
+        )
```

Comment:
Hmm but you are not using `SWIFT_PROJECT_NAME` anymore. My proposal was:

* if SWIFT_PROJECT_NAME is set: use it (that's what we'll want at the end)
* but if SWIFT_PROJECT_NAME is not set and SWIFT_TENANT_NAME is set, it means people are still using the old name for the variable: use it so nothing is broken, but log a message so people know they have to change it

Happy to discuss about this with you on a call if you need